app/settings.py

```diff
@@ -0,0 +1,29 @@
+from os import environ
+
+
+class Settings:
```

Comment:
nie robiłbym z tego klasy tylko normalnie same varsy
```suggestion
from os import environ

EXPIRE_DAYS = int(environ.get('EXPIRE_DAYS', 30))
```

i używać tego 

```
import settings

settings.EXPIRE_DAYS
```

klasa w sumie nie ma żadnej wartości tutaj. ale jak już chcesz mieć klasę to po prostu możesz robić 
`Settings.EXPIRE_DAYS` zamiast `Settings().EXPIRE_DAYS` żeby za każdym razem nie inicjalizować nowego obiektu